src/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css

```diff
@@ -116,7 +116,7 @@ html[data-theme="dark"] {
   * layout
   */
 
-  --pst-color-link: #0088CC;
+  --pst-color-link: #0077B3;
```

Comment:
We shouldn't change this, #0077B3 also fails accessibility guidelines in dark mode.